tests/backuptestTriggers.go

```diff
@@ -404,6 +496,16 @@ func getGlobalBucketName(provider string) string {
 	return bucketName
 }
 
+func getGlobalLockedBucketName(provider string) string {
+	switch provider {
```

Comment:
You don't need the switch case? Because locked bucket is only possible with aws s3 for at least now. You can change this to a if.

Reply:
Updated the code to if , change has been made to original method in backup_basic_test.go line 50 as well.